pkg/providers/launchtemplate/suite_test.go

```diff
@@ -278,12 +274,24 @@ var _ = Describe("LaunchTemplates", func() {
 					ImageId:      aws.String("ami-123"),
 					Architecture: aws.String("x86_64"),
 					CreationDate: aws.String("2022-08-15T12:00:00Z"),
+					Tags: []*ec2.Tag{
```

Comment:
Why do we need to add tags to all the test AMIs now?

Reply:
Removed them and opted for a more permissive selector